docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1295 @@
+{
```

Comment:
<p>Please use <code>pathlib.Path</code>  instead of <code>os</code>.  Always. <a href="https://docs.python.org/3/library/pathlib.html" target="_blank">https://docs.python.org/3/library/pathlib.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Changed it!